tests/trainer/properties/test_loggers.py

```diff
@@ -52,6 +54,10 @@ def test_trainer_loggers_setters():
     logger2 = CustomLogger()
     logger_collection = LoggerCollection([logger1, logger2])
     logger_collection_2 = LoggerCollection([logger2])
+    logger_collection_dep = "Using `trainer.logger` when Trainer is configured to use multiple loggers."
+    " `LoggerCollection` is deprecated in v1.6 in favor of `trainer.loggers` and will be removed in v1.8."
+    " This behavior will change in v1.8 such that `trainer.logger` will return the first"
+    " logger in `trainer.loggers`."
```

Comment:
Don't need to match the full string, `match=...` takes a regex

```suggestion
    logger_collection_dep = "LoggerCollection` is deprecated in v1.6"
```